docs/modules/ROOT/pages/tutorials/tutorial.adoc

```diff
@@ -298,32 +306,49 @@ Let's start with the WordPress pod:
 TIP: In K9s you can open a shell in a pod by selecting that pod and then hitting the kbd:[S] key.
 
 [start=2]
-. Replace the files inside the pod with the ones from the restore:
+. Replace the files inside the pod with the ones from the restore. Using rsync here simplifies things by only copying files that are different from the target and also deleting any additional malicious files an attacker may have added:
 
 ....
-$ cd /var/www/html # or whatever value has `mountPath`
-$ mv data /
-$ rm -r *
-$ cp -R /data/wordpress-pvc/* .
+# apt update && apt install rsync
+# rsync -a --delete /restore/ /var/www/html/
 ....
 
 [start=3]
 . Since it is a PHP application, nothing else is needed for this pod.
 
-In the case of the MariaDB pod, the application must be restarted after restoring the files:
+=== Restoring the database dump
+
+In the case of the MariaDB pod, this backup annotation in the MariaDB deployment instructed K8up to create a database dump as text and store it in MinIO:
+
+[source,yaml]
+....
+spec:
+  template:
+    metadata:
+      annotations:
+        k8up.syn.tools/backupcommand: appuio.ch/backupcommand: /bin/bash -c 'mysqldump -uroot -p"${MARIADB_ROOT_PASSWORD}" --all-databases'
+....
+
+Thus the restoration procedure is different. 
+Instead of restoring files to a PVC, we can instead connect to the running database in the pod and restore directly from the dump.
 
 [start=4]
-. Follow the same logic as previously:
+. Get the snapshot ID for the database dump and save it to your local machine as `backup.sql`:
 
 ....
-$ cd /var/lib/mysql # Attention! Different path!
-$ mv data /
-$ rm -r *
-$ cp -R /data/mariadb-pvc/* .
+$ export SNAPSHOT_ID=$(restic snapshots --json --last --path /default-mariadb | jq -r '.[0].id')
+$ restic dump "${SNAPSHOT_ID}" /default-mariadb > backup.sql
 ....
 
 [start=5]
-. Delete the MariaDB pod and a new one will be immediately created due to the deployment policy (see `mariadb/deployment.yml`): `kubectl delete pod mariadb-XXXXXXXXXX-xxxx`
+. Now connect to the MariaDB pod, copy the dump and a simple restore script to the pod and execute it there:
+
+....
```

Comment:
You can indent the code block and make it part of the numbering by adding the + between. Maybe you can then avoid having to specifically set the numbering via `[start=x]` in the next item
```suggestion
. Now connect to the MariaDB pod, copy the dump and a simple restore script to the pod and execute it there:
+
....
```